bindings/python/setup.py

```diff
@@ -108,58 +114,56 @@ def build_libraries(self, libraries):
         if SYSTEM in ("win32", "cygwin"):
             # if Windows prebuilt library is available, then include it
             if is_64bits and os.path.exists(PATH_LIB64):
-                SETUP_DATA_FILES.append(PATH_LIB64)
+                shutil.copy(PATH_LIB64, "capstone")
                 return
             elif os.path.exists(PATH_LIB32):
-                SETUP_DATA_FILES.append(PATH_LIB32)
+                shutil.copy(PATH_LIB32, "capstone")
                 return
 
         # build library from source if src/ is existent
         if not os.path.exists('src'):
             return
 
-        try:
```

Comment:
Because if something goes wrong it will install a broken package - the library will not be there and it wont work. Its better to know that installation failed than to install a broken package.